src/materialized/src/bin/materialized.rs

```diff
@@ -428,29 +439,17 @@ fn handle_panic(panic_info: &PanicInfo) {
         },
     };
 
-    let backtrace = Backtrace::new();
-
-    let crash_message = format!(
+    log::error!(target: "panic", "{}: {}\n{:?}", thr_name, msg, Backtrace::new());
+    eprintln!(
```

Comment:
does it make sense to make this `eprintln` part of the log statement above, so that it goes to both the  logfile and stderr?

Reply:
It makes slightly more sense to me to have it only go to stderr, but I don't feel strongly! Just didn't seem worth the CI cycle.